geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/RemoveAllPRMessage.java

```diff
@@ -532,6 +537,9 @@ public boolean doLocalRemoveAll(PartitionedRegion r, InternalDistributedMember e
         } catch (RegionDestroyedException e) {
           ds.checkRegionDestroyedOnBucket(bucketRegion, true, e);
         } finally {
+          if (rvvLocked) {
+            bucketRegion.unlockRVVForBulkOp();
+          }
```

Comment:
Should you be setting rvvLocked to false?

Reply:
good idea